imagesloaded.js

```diff
@@ -273,6 +273,7 @@ LoadingImage.prototype.check = function() {
 
   // If none of the checks above matched, simulate loading on detached element.
   this.proxyImage = new Image();
+  this.proxyImage.crossOrigin = this.img.crossOrigin;
```

Comment:
is there ever a time we'd not have crossOrigin set and would need to check that it exists before setting it on the 
proxyImage? Only other thing I'd say from a pure aesthetics perspective would be to do this assignment near line 282 when the src is set.

Reply:
`this.img` is an `HTMLImageElement`, these always have the `crossOrigin` attribute,  if none is specifically set, it will return `null`, which won't throw an error and has no effect.

I'd like to leave this where it is, because order of operations specifically around setting `crossOrigin` and `src` is very particular, and there are event listeners added around this, I wanted to keep this solution as close as possible to the solution it was originally taken from in the PR (linked in the description). I don't believe adding event listeners should make any difference but didn't feel like testing those theories.